ec2api/api/common.py

```diff
@@ -343,6 +347,7 @@ def describe(self, context, ids=None, names=None, filter=None):
         self.names = set(names or [])
         self.items = self.get_db_items()
```

Comment:
make sure that self.get_db_items() returns list only. If it returns None (at some situation (don't know)) below append operation will fails.